analysis/tc_tomcat.go

```diff
@@ -20,115 +20,712 @@ var Tomcat = TC{
 		"linux",
 	},
 	Analysis: api.Analysis{
-		Effort: 1,
-		Issues: []api.Issue{
-			{
-				Category:    "cloud-mandatory",
-				Description: "Trivial change or 1-1 library swap",
-				Effort:      1,
-				RuleSet:     "DiscoverHardcodedIPAddressRuleProvider",
-				Rule:        "DiscoverHardcodedIPAddressRuleProvider",
-				Incidents: []api.Incident{
-					{
-						File:    "persistence.properties",
-						Line:    2,
-						Message: "Hard-coded IP: 169.60.225.216When migrating environments, hard-coded IP addresses may need to be modified or eliminated.",
-					},
-				},
-			},
-			{
-				Category:    "information",
-				Description: "Info",
-				Effort:      0,
-				RuleSet:     "database",
-				Rule:        "database-03200",
-				Incidents: []api.Incident{
-					{
-						File:    "pom.xml",
-						Line:    0,
-						Message: "The application embeds Spring Data JPA",
-					},
-				},
-			},
-			{
-				Category:    "information",
-				Description: "Info",
-				Effort:      0,
-				RuleSet:     "mvc",
-				Rule:        "mvc-01220",
-				Incidents: []api.Incident{
-					{
-						File:    "pom.xml",
-						Line:    0,
-						Message: "The application embeds a Spring MVC library.",
-					},
-				},
-			},
-			{
-				Category:    "information",
-				Description: "Info",
-				Effort:      0,
-				RuleSet:     "embedded-framework",
-				Rule:        "embedded-framework-08400",
-				Incidents: []api.Incident{
-					{
-						File:    "pom.xml",
-						Line:    0,
-						Message: "The application embeds the Spring Web framework.",
-					},
-				},
-			},
-			{
-				Category:    "information",
-				Description: "Info",
-				Effort:      0,
-				RuleSet:     "observability",
-				Rule:        "observability-0100",
-				Incidents: []api.Incident{
-					{
-						File:    "pom.xml",
-						Line:    0,
-						Message: "The application embeds Spring Boot Actuator.",
-					},
-				},
-			},
-			{
-				Category:    "information",
-				Description: "Info",
-				Effort:      0,
-				RuleSet:     "technology-usage-javaee",
-				Rule:        "javaee-technology-usage-00120",
-				Incidents: []api.Incident{
-					{
-						File:    "OrderManagementAppInitializer.java",
-						Line:    0,
-						Message: "The application uses Java Servlets",
-					},
-				},
-			},
-			{
-				Category:    "information",
-				Description: "Info",
-				Effort:      0,
-				RuleSet:     "DiscoverMavenProjectsRuleProvider",
-				Rule:        "DiscoverMavenProjectsRuleProvider_1",
-				Incidents: []api.Incident{
-					{
-						File:    "pom.xml",
-						Line:    0,
-						Message: "Maven Project Object Model (POM) File",
-					},
+		Effort: 0,
```

Comment:
effort 0 doesn't make sense for this app. Something must be wrong

Reply:
Fixed that, thanks